arroyo/processing/strategies/collect.py

```diff
@@ -32,7 +21,7 @@ def __init__(
 
         self.__created = time.time()
         self.__length = 0
-        self.__offsets: MutableMapping[Partition, OffsetRange] = {}
+        self.__offsets: MutableMapping[Partition, Position] = {}
```

Comment:
Any functional reason to replace `OffsetRange` with `Position` or is it just a simplification?

Reply:
No. Lo was never accessed